reference/tools/meson/mesontoolchain.rst

```diff
@@ -151,6 +151,24 @@ The ``wrap_mode: nofallback`` is defined by default as a project option, to make
 
 Note that in this case, Meson might be able to find dependencies in "wraps", it is the responsibility of the user to check the behavior and make sure about the dependencies origin.
 
+subproject_options
+^^^^^^^^^^^^^^^^^^
+
+This attribute allows defining Meson subproject options:
+
+.. code:: python
+
+    def generate(self):
+        tc = MesonToolchain(self)
+        tc.subproject_options["SUBPROJECT"] = [{'MYVAR': 'MyValue'}]
+        tc.generate()
+
+This is translated to:
+
+- One subproject ``SUBPROJECT`` along with options definition for ``MYVAR`` in ``conan_meson_native.ini`` or ``conan_meson_cross.ini`` file.
+
+Note that in contrast to project_options, subproject_options is a dictionary of lists of dictionaries. This is because Meson allows multiple subprojects, and each subproject can have multiple options.
```

Comment:
```suggestion
Note that in contrast to ``project_options``, ``subproject_options`` is a dictionary of lists of dictionaries. This is because Meson allows multiple subprojects, and each subproject can have multiple options.
```